packages/salesforcedx-utils-vscode/src/telemetry/cliConfiguration.ts

```diff
@@ -5,31 +5,12 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { which } from 'shelljs';
```

Comment:
Moved this to the telemetry module cause it depends on `ConfigUtil` which has a `vscode` dependency. When bundling the code, it was causing unit tests in other modules to fail since they couldn't resolve that dependency. We might move this and configUtil back to the cli module once we sort out those type of problems.